Library/Homebrew/formulary.rb

```diff
@@ -306,7 +308,7 @@ def versioned_formulae_names
       end
     end
 
-    klass.loaded_from_api = true
+    T.cast(klass, T.class_of(Formula)).loaded_from_api = true
```

Comment:
Can you move this to where `klass` is defined instead?

Reply:
This is the correct place for the cast:
- `klass` is assigned the result of a 150+ line block, which is clumsy to wrap in a cast
- casts should be placed where needed, except to avoid repetitive casts. Otherwise, a change that removes this line is likely to leave an unnecessary cast elsewhere.